File: app/src/main/java/com/hjc/base/ui/video/fragment/PictureFragment.java
package com.hjc.base.ui.video.fragment;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;

import com.hjc.base.R;
import com.hjc.base.databinding.FragmentPitureBinding;
import com.hjc.base.viewmodel.CommonViewModel;
import com.hjc.library_base.fragment.BaseFragment;

/**
 * @Author: HJC
 * @Date: 2019/7/26 10:42
 * @Description: Tab1
 */
public class PictureFragment extends BaseFragment<FragmentPitureBinding, CommonViewModel> {

    public static PictureFragment newInstance() {
        return new PictureFragment();
    }

    @Override
    public int getLayoutId() {
        return R.layout.fragment_piture;
    }

    @Override
    public CommonViewModel createViewModel() {
        return null;
    }

    @Override
    public void initData(@Nullable Bundle savedInstanceState) {

    }

    @Override
    public void addListeners() {
        mBindingView.setOnClickListener(this);
    }

    @Override
    public void onSingleClick(View v) {

    }

}

File: module-other/src/main/java/com/hjc/module_other/ui/video/child/PictureFragment.kt
package com.hjc.module_other.ui.video.child

import android.os.Bundle
import android.view.View
import com.hjc.library_base.fragment.BaseFragment
import com.hjc.library_common.utils.image.ImageManager
import com.hjc.library_common.viewmodel.CommonViewModel
import com.hjc.module_other.R
import com.hjc.module_other.databinding.OtherFragmentPictureBinding

/**
 * @Author: HJC
 * @Date: 2019/7/26 10:42
 * @Description: 展示图片
 */
class PictureFragment : BaseFragment<OtherFragmentPictureBinding, CommonViewModel>() {

    companion object {
        fun newInstance(imageUrl: String): PictureFragment {
            val fragment = PictureFragment()
            val bundle = Bundle()
            bundle.putString("imageUrl", imageUrl)
            fragment.arguments = bundle
            return fragment
        }
    }

    override fun getLayoutId(): Int{
        return  R.layout.other_fragment_picture
    }

    override fun createViewModel(): CommonViewModel? {
        return null
    }

    override fun initData(savedInstanceState: Bundle?) {
        arguments?.let {
            val imageUrl = it.getString("imageUrl", "")
            ImageManager.loadImage(mBindingView.ivCover, imageUrl)
        }
    }

    override fun addListeners() {

    }

    override fun onSingleClick(v: View?) {

    }

}